be/src/vec/columns/column_object.cpp

```diff
@@ -436,8 +436,7 @@ ColumnPtr ColumnObject::index(const IColumn& indexes, size_t limit) const {
 }
 
 bool ColumnObject::Subcolumn::check_if_sparse_column(size_t num_rows) {
-    constexpr static size_t s_threshold_rows_estimate_sparse_column = 1000;
-    if (num_rows < s_threshold_rows_estimate_sparse_column) {
+    if (num_rows < config::variant_threshold_rows_to_estimate_sparse_column) {
```

Comment:
If there are many columns in the variant_threshold_rows_to_estimate_sparse_column rows, will it cause problem.

Reply:
only sparse column will be affected， others will not